src/Components/Button/styles.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components/native';
+import { fonts } from '../../Constants/constants';
```

Comment:
Configura absolute path. Caso acabe ocorrendo um nesting de pastas, pode acabar ficando caótico.
```suggestion
import { fonts } from 'Constants/constants';
```